core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -82,9 +86,13 @@
     private EncodedValue highwayEncoder;
     private EncodedValue transportModeEncoder;
     private EncodedValue accessEncoder;
+    private static int NOT_ACCESSIBLE_KEY;
```

Comment:
Somehow I cannot comment on your comment below.

> Can it be removed?

Yes, but it is used two times in the code. This would most probably lead to duplicated code. So I wouldn't want to do this, only if we have to?